libsolidity/analysis/NameAndTypeResolver.cpp

```diff
@@ -187,6 +187,19 @@ vector<Declaration const*> NameAndTypeResolver::nameFromCurrentScope(ASTString c
 	return m_currentScope->resolveName(_name, true, _includeInvisibles);
 }
 
+Declaration const* NameAndTypeResolver::nameFromEnclosingScope(ASTString const& _name) const
+{
+	DeclarationContainer const* enclosingContainer = m_currentScope->enclosingContainer();
+	if (!enclosingContainer)
+		return nullptr;
+
+	vector<Declaration const*> candidates = enclosingContainer->resolveName(_name, true);
+	if (candidates.size() == 1)
+		return candidates.front();
+	else
+		return nullptr;
```

Comment:
iiuc, there can be multiple `candidates.` Why is a `nullptr` returned when `candidates.size() > 1`

Reply:
Same as in `pathFromCurrentScope`, returning `nullptr` here indicates that identifier is not unique.